src/MudBlazor.Markdown/Resources/MudBlazor.Markdown.css

```diff
@@ -34,7 +34,10 @@ pre code.hljs {
 	color: var(--mud-palette-text-secondary);
 	background-color: var(--mud-palette-drawer-background);
 	padding: 0.25em 1em;
-	margin: 0.5em 0;
+	margin: 0.5em 0 1.25em;
```

Comment:
Is there any reasoning behind setting the horizontal margin to 1.25em? It break alignment of the block element.
![image](https://github.com/MyNihongo/MudBlazor.Markdown/assets/47413092/5a1408c3-12d6-4ba3-be58-d6988f7d9f90)

Reply:
1.25em is actually setting the bottom margin :)

> /* top | left and right | bottom */
> margin: 1em auto 2em;

source: https://developer.mozilla.org/en-US/docs/Web/CSS/margin

I can't replicate that behaviour of horizontal margin occurring - maybe clear cache and hard reload?

Reply:
Sorry, I looked at it incorrectly, treated it as `margin: 0.5em 1.25em;`
Then I have no more questions. On github there seems to be some bottom margin as well, so this change is all right/